app/assets/javascripts/discourse/tests/acceptance/keyboard-shortcuts-test.js

```diff
@@ -41,3 +42,67 @@ acceptance("Keyboard Shortcuts", function (needs) {
     assert.equal(currentURL(), "/t/keyboard-shortcuts-are-awesome/27331");
   });
 });
+
+acceptance("Keyboard Shortcuts - Authenticated Users", function (needs) {
+  let resetNewCalled = 0;
+  let markReadCalled = 0;
```

Comment:
Don't we have to reset this after every test run? :thinking:

Reply:
Ah you're right that is a goof, forgot that these `acceptance` blocks are like the definitions of the test, will change

Reply:
Oo this didn't cause any problems with the tests you added because each test asserts against one of the two variables only.

Reply:
Oh well, still made it neater anyway!